spring-integration-file/src/test/java/org/springframework/integration/file/remote/RemoteFileTestSupport.java

```diff
@@ -162,15 +172,24 @@ public void recursiveDelete(File file) {
 			File[] files = file.listFiles();
 			if (files != null) {
 				for (File fyle : files) {
+					logger.info("Deleting: " + fyle + " in " + testName.getMethodName());
 					if (fyle.isDirectory()) {
 						recursiveDelete(fyle);
 					}
 					else {
-						fyle.delete();
+						if (!fyle.delete()) {
+							logger.error("Couldn't delete: " + fyle + " in " + testName.getMethodName());
+						}
 					}
 				}
 			}
-			file.delete();
+			logger.info("Deleting: " + file + " in " + testName.getMethodName());
```

Comment:
I figured it won't hurt leaving these at INFO level - in case we have problems in future.